lib/broadcast.js

```diff
@@ -7,9 +7,85 @@
 * {@link OpenTok#startBroadcast OpenTok.startBroadcast()} method.
 *
+* @property {String} id
+*   The broadcast ID.
+* @property {String} sessionId
+*   The session ID of the OpenTok session associated with this broadcast.
+* @property {String} projectId
+*   The API key associated with the broadcast.
 * @property {Number} createdAt
-*   The time at which the archive was created, in milliseconds since the UNIX epoch.
+*   The time at which the broadcast was created, in milliseconds since the UNIX epoch.
+* @property {String} resolution
+*   The resolution of the broadcast: one of the following:
+*   <ul>
+*     <li>"640x480"</li>
+*     <li>"1280x720"</li>
+*     <li>"1920x1080"</li>
+*     <li>"480x640"</li>
+*     <li>"720x1280"</li>
+*     <li>"1080x1920"</li>
+*   </ul>
+*   You may want to use a portrait aspect ratio for broadcasts that include video streams from
+*   mobile devices (which often use the portrait aspect ratio). This property is optional.
+* @property {Object} broadcastUrls
+*   An object containing details about the HLS and RTMP broadcasts.
+*   <p>
+*   <ul>
+*     <li>
+*       <code>hls</code> (String) &mdash; If you specified an HLS endpoint, the object includes
+*       an hls property, which is set to the URL for the HLS broadcast. Note this HLS broadcast
+*       URL points to an index file, an .M3U8-formatted playlist that contains a list of URLs to
+*       .ts media segment files (MPEG-2 transport stream files). While the URLs of both the
+*       playlist index file and media segment files are provided as soon as the HTTP response
+*       is returned, these URLs should not be accessed until 15 – 20 seconds later, after the
+*       initiation of the HLS broadcast, due to the delay between the HLS broadcast and the live
+*       streams in the OpenTok session.
+*       See https://developer.apple.com/library/ios/technotes/tn2288/_index.html for more
+*       information about the playlist index file and media segment files for HLS.
+*     </li>
+*     <li>
+*       <code>rtmp</code> (Object Array) &mdash; If you specified RTMP stream endpoints,
+*       the object includes an rtmp property. This is an array of objects that include
+*       information on each of the RTMP streams.
+*       Each of these objects has the following properties:
+*       <ul>
+*         <li><code>id</code> The ID you assigned to the RTMP stream</li>
+*         <li><code>serverUrl</code> The server URL</li>
+*         <li><code>streamName</code> The stream name</li>
+*         <li><code>status</code> The status of the stream</li>
+*       </ul>
+*     </li>
+*   </ul>
+* @property {Number} maxDuration
+*   Maximum time allowed for the broadcast, in seconds.
+*   After this time, the broadcast will be stopped automatically, if it is still started
+*   Value can be the following:
+*   <ul>
+*     <li>"minimum" -- 60
+*     <li>"maximum" -- 36000
+*     <li>"default" -- 7200
+*   </ul>
```

Comment:
The value is a number, not a string. Should this be:

```suggestion
You can set the maximum duration to a value from 60 (60 seconds) to
36000 (10 hours). The default maximum duration is 4 hours (14,400 seconds).
```